packages/thirdweb/src/chainsaw/urls.ts

```diff
@@ -0,0 +1,54 @@
+import { getThirdwebBaseUrl } from "../utils/domains.js";
+
+export function getChainsawV2Url(): string {
```

Comment:
consider operating on URL with all of this (see other comment way above)

Reply:
Done in [8a11f63](https://github.com/thirdweb-dev/js/pull/4004/commits/8a11f633241771a4ae31adbe4c4b61691258a080) and [534d099](https://github.com/thirdweb-dev/js/pull/4004/commits/534d099ac74bd8e047a5a2a39c3cc4f60af401ed)